packages/billing/src/tools/billingCentrifuge.spec.ts

```diff
@@ -24,19 +42,19 @@ describe('billingCentrifuge', () => {
             const recipient3 = createTestRecipient()
             const recipient4 = createTestRecipient()
 
-            const distribution = [
-                { recipient: recipient1, order: 0, vor: true },
-                { recipient: recipient2, order: 0 },
-                { recipient: recipient3, order: 2, vor: true },
-                { recipient: recipient4, order: 1 },
+            const distribution: TDistributionItem[] = [
+                { recipient: recipient1, order: 0, amount: '', vor: true },
```

Comment:
why empty string is even a valid option here?

Reply:
Same as `0`

Reply:
Idk, seems like bad design a bit

Remember, when building a fin-tech utils it's better to throw explicit error, when something is not in format, than trying to interpret invalid input somehow...

Reply:
Fixed test